ChromeDriverAmazonMusic.py

```diff
@@ -8,16 +8,15 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
+from FuzzyMatcher import *
 
 
 class ChromeDriverAmazonMusic:
     driver = None
     playlist_name = None
     asset_list = None
-    status = None
-    # only for testing
-    email = "email"
-    password = "password"
+    status_matched = None
+    status_failed = None
 
     def __init__(self, test_mode=False, email=None, password=None, playlist=None, asset_list=None):
```

Comment:
What will be the behaviour for the case test_mode=True and other arguments are None?

Reply:
The driver isn't supposed to run with such a config legally.....
-It can be either (test_mode=false, others none) as then email/pass/playlist name are expected from the user, asset list from the other Handlers. (Once we have a central file that synchronises all operations)

-Or (test_mode=true, email, password) and playlist/asset_list optional only.